libr/core/vmenus.c

```diff
@@ -83,13 +83,13 @@ static char *colorize_asm_string(RCore *core, const char *buf_asm, int optype) {
 		if (!scol2) {
 			scol2 = strdup ("");
 		}
-		source = malloc (strlen(scol1) + strlen(scol2) + 2 + 1); // reuse source variable
+		source = r_str_newf ("%s||%s", scol1, scol2); // reuse source variable
 		sprintf (source, "%s||%s", scol1, scol2);
```

Comment:
if you do r_str_newf there's no need to sprintf after that, also if you use r_str_get() you can remove the checks above where if (!scol2) scol2=strdup("")

Reply:
i know this code is not yours, just pointing it out. if you can fix it now, it will be great, if not, i'll merge and fix later. thanks!